main.bundle.js

```diff
@@ -52,19 +52,48 @@
 
 	var _currentWeather2 = _interopRequireDefault(_currentWeather);
 
+	var _backgroundImage = __webpack_require__(6);
+
+	var _backgroundImage2 = _interopRequireDefault(_backgroundImage);
+
 	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
-	// This file is in the entry point in your webpack config.
-	$('form').on('submit', function (event) {
+	$(document).ready(function () {
+	  $('#location-form').on('submit', getLocationInfo);
+	  $('#sign-up').on('submit', registerUser);
+	}); // This file is in the entry point in your webpack config.
+
+
+	var registerUser = function registerUser(event) {
 	  event.preventDefault();
-	  var location = $('input').val().toLowerCase();
-	  $.ajax({
-	    type: "GET",
-	    url: "http://localhost:3000/api/v1/forecast?location=" + location
-	  }).then(setCurrentWeather).catch(function (error) {
-	    console.log('Error: ' + error);
+	  $('.errors').text('');
+	  var email = $('input')[0].value;
+	  var password = $('input')[1].value;
+	  var confPassword = $('input')[2].value;
+	  if (!password || !confPassword || password !== confPassword) {
+	    displayError("Error: Please check passwords");
+	  } else {
+	    createUser(email, password, confPassword);
```

Comment:
This is where you would instantiate your new instance of the class User if you decide to go the route of separating it into a class.